src/features/common/Layout/Navbar/index.tsx

```diff
@@ -1,30 +1,59 @@
+import Link from 'next/link';
 import { useRouter } from 'next/router';
 import React from 'react';
-import tenantConfig from '../../../../../tenant.config';
+import i18next from '../../../../../i18n';
 import Me from '../../../../../public/assets/images/navigation/Me';
 import MeSelected from '../../../../../public/assets/images/navigation/MeSelected';
+import tenantConfig from '../../../../../tenant.config';
 import { ThemeContext } from '../../../../theme/themeContext';
-import i18next from '../../../../../i18n';
-import getImageUrl from '../../../../utils/getImageURL';
 import themeProperties from '../../../../theme/themeProperties';
-import Link from 'next/link';
-import GetNavBarIcon from './getNavBarIcon';
+import getImageUrl from '../../../../utils/getImageURL';
 import { UserPropsContext } from '../UserPropsContext';
-import { lang_path } from '../../../../utils/constants/wpLanguages';
+import GetNavBarIcon from './getNavBarIcon';
+import GetSubMenu from './getSubMenu';
 
+// used to detect window resize and return the current width of the window
+const useWidth = () => {
+  const [width, setWidth] = React.useState(0); // default width, detect on server.
+  const handleResize = () => setWidth(window.innerWidth);
+  React.useEffect(() => {
+    window.addEventListener('resize', handleResize);
+    handleResize();
+    return () => window.removeEventListener('resize', handleResize);
+  }, [handleResize]);
+  return width;
+};
 const { useTranslation } = i18next;
 const config = tenantConfig();
 export default function NavbarComponent(props: any) {
   const { t, ready, i18n } = useTranslation(['common']);
   const router = useRouter();
+  const lang_path = {
```

Comment:
Please use https://github.com/Plant-for-the-Planet-org/planet-webapp/blob/develop/src/utils/constants/wpLanguages.ts which @sarveshpro added for his implementation of language dependent Wordpress links (we need to have a common software architecture and do not build stuff twice).